apps/studio-next/package.json

```diff
@@ -9,25 +9,80 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@codemirror/state": "^6.4.1",
```

Comment:
@KhudaDad414 why codemirror is revered. It was added by @Shurtu-gal in https://github.com/asyncapi/studio/pull/997

Reply:
I don't think it's a great idea to push for the CodeMirror at this stage. Migrating to CodeMirror would make the task much much more challenging and this task is already huge.

The CodeMirror was added as a separate code editor and it wasn't configured with the codebase in any shape or form. I imagine integrating a new code editor with our codebase would also be challenging.

here is what I am suggesting: the code is there in the history, let's go ahead with Monaco we can swap the code editor as part of [the bounty issue](https://github.com/asyncapi/studio/issues/684).

TLDR; migration is already complicated enough let's switch to CodeMirror after we migrate to NEXT.